test/test_meta.py

```diff
@@ -241,6 +241,7 @@ def test_tensor_outlives_converter(self):
     aten.maximum.default,
     aten.minimum.default,
     aten.mul.Tensor,
+    aten.narrow_copy.default,
```

Comment:
Had to skip due to the following new failure. This wasn't failing before because the `out` overload [wasn't defined for cuda](https://github.com/pytorch/pytorch/pull/49502) (not sure why, the original PR mentions cuda failures, but nothing concrete) and this test only runs on cuda. This test was also [added recently](https://github.com/pytorch/pytorch/pull/87849), so I'm not sure whether the failures are expected (there's quite a few skips).

```
TestMetaCUDA.test_dispatch_symbolic_meta_outplace_all_strides_narrow_copy_cuda_float32
[...]
RuntimeError: output 0: meta disagrees with real impl:
aten.narrow_copy.default(
  tensor(..., device='meta', size=(5, 5, 5)) stride=(1, 5, 25),
  1,
  2,
  2,

) = (
  tensor(..., device='meta', size=(5, 2, 5)) stride=(1, 5, 10)
)
but real stride was (10, 5, 1)
```

Also: an issue with the out variant of `narrow_copy` was reported [here](https://github.com/pytorch/pytorch/issues/84577). I cannot reproduce after making these changes.

Reply:
FWIW, `narrow_copy` should be implemented from the decomposition of `narrow` via https://github.com/pytorch/pytorch/pull/87278. You can add this to this PR if you feel like it, as it should be a small change.

Reply:
Note: the issue with strides disappeared after switching to `_make_copy_from_view`.